src/main/java/io/github/classgraph/MethodInfo.java

```diff
@@ -318,6 +324,24 @@ public String getTypeSignatureOrTypeDescriptorStr() {
         }
     }
 
+    public ClassInfoList getThrownExceptions() {
```

Comment:
When a thrown exception is a generic type variable, the `Exceptions` attribute refers to the bound type, which I think makes sense to return here?

Reply:
Yes, I think so, or if all else fails and the type variable cannot be resolved or does not have a supertype (e.g. `E extends IOException`), just return `Exception` as the concrete type, I think.

It's a shame Java doesn't have union types, since it would be nice to return `TypeVariable` or `ClassInfo` depending on whether or not the class variable is unresolved.

Also you can discard type parameters like `T` in `MyException<T>`. If anyone needs those, they can use the existing method to get the generic `throws` type signature.

Reply:
> Also you can discard type parameters like T in MyException<T>

I believe that is how this functions currently.